yt/yql/tests/test_simple.py

```diff
@@ -105,6 +105,58 @@ def test_calc(self, query_tracker, yql_agent):
         """, [[{"column0": 1}], [{"column0": 2}], [{"a": 1, "b": 2}]])
 
 
+class TestYqlAgentBan(TestQueriesYqlBase):
+    NUM_YQL_AGENTS = 1
+    YQL_AGENT_DYNAMIC_CONFIG = {"state_check_period": 2000}
+
+    def query_fails():
```

Comment:
a bit weird name for a method, looks like a verb is missing

Reply:
I used "fail" as a verb, like "complete" in the next one

Reply:
next one also looks like a place for improvement)

method names usually start with verb

I would use names like  
 * _test_query_fails / _test_query_completes (with "test" like in test_and_set, not a separate integration tets)
 * check_query_fails / check_query_completes
 * run_query_fail_check / run_query_complete_check